steps/expression_processor.go

```diff
@@ -21,7 +21,52 @@ func RegisterExpression(b reg.ProcessorRegistry) {
 			return add_expression(p, params, false)
 		},
 		"Execute the given expression on every sample").
-		Required("expr", reg.String())
+		Required("expr", reg.String(), "Allows arithmetic and boolean operations. Can also perform them on sample fields "+
+			"(e.g. \"field1\" [+,-,*,/] \"field2\").",
```

Comment:
This example does not convey the real meaning - the syntax with quotes (e.g. `"field`") evaluates to a string, and not to the field value. Arithmetical operations are not provided for strings (except for `+`).

Reply:
Done